integration/setup/setup.go

```diff
@@ -70,7 +69,12 @@ func installResources(ctx context.Context, config Config) error {
 		for _, crdName := range crds {
 			config.Logger.Debugf(ctx, "ensuring %#q CRD exists", crdName)
 
-			err := config.K8sClients.CRDClient().EnsureCreated(ctx, crd.LoadV1("application.giantswarm.io", crdName), backoff.NewMaxRetries(7, 1*time.Second))
+			crd, err := config.CRDGetter.LoadCRD(ctx, "application.giantswarm.io", crdName)
```

Comment:
As you see, it supports almost the same convention we used it from apiextensions.